javascript/javascript1/week2/week2-guzide/student-manager/StudentManager.js

```diff
@@ -0,0 +1,37 @@
+const class07Students = [];
+
+function getNumberOfStudents() {
+  return class07Students.length;
+}
+
+function addStudentToClass(studentName) {
+  if (studentName == "") {
+    console.log("You can't leave this area empty");
+    return;
+  }
+
+  if (studentName === "Queen Mary") {
```

Comment:
I think the specification for this just mentioned "Quenn" as a special student name :)

Reply:
`"Queen Mary"`is changed with `"Queen"`.